README.md

```diff
@@ -17,9 +17,14 @@
 
 ## Introduction
 
-[Augury](https://augury.angular.io/) is a Google Chrome Dev Tools extension for debugging Angular 2+ applications.
+[Augury](https://augury.angular.io/) is a Chrome & Firefox Developer Tools extension for debugging Angular 2+ applications.
+
+You can install the extension from:
+
+- [Chrome Store](https://chrome.google.com/webstore/detail/augury/elgalmkoelokbchhkhacckoklkejnhcd)
+- [Firefox Add-ons](https://addons.mozilla.org/en-US/firefox/addon/angular-augury)
```

Comment:
This won't be working in this release, but I don't see it being a big issue at this point having it here.